### ISSUE TITLE
An option to heal all powered nodes

### ISSUE THREAD

**Issue by ColColonCleaner** [AlCalzone/node-zwave-js#5372]
**Is your feature request related to a problem? Please describe.**
When calling for a network heal it includes every node in the network. I don't want to go around to the 30 battery nodes every time so it can mark the heal as 'complete'. Whenever a heal is waiting on battery nodes to connect the heal can't be ended. The 'stop healing' option doesn't seem to do anything as the nodes still have their loading masks in the heal column after stopping the heal. And after stopping a network heal before it's marked 'complete' by going through every node, individual heals on nodes get automatically marked failed without trying. I know this happens when doing two individual heals at once, it looks like they can't be queued so i need to wait for one to finish before calling the next one but that's fine for ones and twos.

If this is just a bug where stopping the heal doesn't actually stop things and mark it complete then i can create a separate issue for that. The only way I've found to get it to stop fully is by rebooting the VM running HA.

**Describe the solution you'd like**
Regardless of whether it's a bug that led me down this line of thinking, having an option to heal all powered nodes so the user can try getting the backbone of their network solid before doing an all inclusive heal would be great.

**Describe alternatives you've considered**
I've currently been using the save settings feature on ZUI for something similar to this. I use it very frequently. Just going into settings and hitting save bounces the driver and calls for all of the powered nodes to communicate. Essentially using the raw communication from all those powered nodes as a way to heal things. Not sure if I'm shooting myself in the foot by doing it this way but I'm sure I'll find out shortly lol. Sometimes I'll do it many times in a row since it only takes about a minute or so for the communication to settle down and finish after the save.

**Additional context**

**Comment by AlCalzone** [comment]
I need to double-check but this could just be a UI issue where Z-UI doesn't update the heal status when aborting the heal.

**Comment by AlCalzone** [comment]
It is => https://github.com/zwave-js/zwave-js-ui/issues/2926

I guess we can add an option to omit battery powered nodes from the heal, but you should be able to achieve the same by just stopping the heal when all mains powered nodes are done. (assuming the UI issue is fixed)

**Comment by ColColonCleaner** [comment]
It would be nice to have the powered nodes option because if i start doing something else and forget to stop the heal once the powered nodes are done the network is essentially unusable. Tons of dead nodes and dropped commands if I forget to stop it lol.

**Comment by ColColonCleaner** [comment]
Are you sure it actually stops the heal when stop is pressed? You said it successfully calls the stop command, but does the heal actually stop?

I say this because I continue to have dead nodes, dropped commands, and laggy interactions if i don't restart the VM after ending a heal early. Regardless of pressing the stop button the network still performs really badly until the VM is bounced.

**Comment by robertsLando** [comment]
Fixed in zui 8.8.3

**Comment by robertsLando** [comment]
BTW @AlCalzone This could easily be done with a driver function snippet.

He could edit the re-interview script to do a heal instead and comment out the last part that interviews sleeping nodes

https://github.com/zwave-js/zwave-js-ui/blob/master/snippets/reinterview-nodes.js

**Comment by AlCalzone** [comment]
@ColColonCleaner Can you capture that problem in a log? I don't have a large network right now, so all I could verify is that the queued commands for battery nodes are discarded.

**Comment by AlCalzone** [comment]
I have a PR up that allows skipping the heal for sleeping devices. However, I'd still like to investigate the issue at hand.

**Comment by ColColonCleaner** [comment]
Here are log files of healing after the latest update to ZUI.

1. Restarted the driver so it was fresh
2. waited for all of the nodes to report and commands/messages to settle down
3. started the heal
4. waited for all of the powered nodes to complete
5. stopped the heal
6. noted that the sleeping nodes were marked skipped, nice!
7. waited a couple minutes
8. tried to interact with some of the nodes
9. Several of them lagged so long they got marked dead, and the rest were very slow to operate. It's basically like the heal was still going.

[z-ui_2023-02-01.log](https://github.com/zwave-js/node-zwave-js/files/10564952/z-ui_2023-02-01.log)
[zwavejs_2023-02-01.log](https://github.com/zwave-js/node-zwave-js/files/10564953/zwavejs_2023-02-01.log)

**Comment by AlCalzone** [comment]
It seems that all of your issues are somehow entangled. The lag you're seeing is caused by spam from the network while the controller is sending and cannot reply, while at the same time causing the controller to take much longer for sending than normal, here for a response to Node 39:
```
2023-02-02T04:53:20.512Z DRIVER » [Node 039] [REQ] [SendDataBridge]
                                  │ source node id:   1
                                  │ transmit options: 0x05
                                  │ callback id:      207
                                  └─[SecurityCCNonceReport]
                                      nonce: 0x7f8bddb85bf93676

... some incoming messages ...

... Node 39 repeats its request a bunch of times
2023-02-02T04:53:21.833Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -72 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:21.835Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:22.451Z SERIAL « 0x010b00a800012702984000b616                                        (13 bytes)
2023-02-02T04:53:22.452Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:22.454Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -74 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:22.456Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:22.536Z SERIAL « 0x010b00a800012702984000b616                                        (13 bytes)
2023-02-02T04:53:22.537Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:22.538Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -74 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:22.539Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:23.186Z SERIAL « 0x010b00a800012702984000aa0a                                        (13 bytes)
2023-02-02T04:53:23.188Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:23.189Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -86 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:23.191Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:23.386Z SERIAL « 0x010b00a800012702984000b717                                        (13 bytes)
2023-02-02T04:53:23.387Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:23.388Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:23.389Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:23.840Z SERIAL « 0x010b00a800012702984000b717                                        (13 bytes)
2023-02-02T04:53:23.842Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:23.843Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:23.845Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:23.925Z SERIAL « 0x010b00a800012702984000b717                                        (13 bytes)
2023-02-02T04:53:23.926Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:23.927Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:23.928Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:24.392Z SERIAL « 0x010b00a800012702984000b616                                        (13 bytes)
2023-02-02T04:53:24.395Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:24.397Z DRIVER « [Node 039] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -74 dBm
                                  └─[SecurityCCNonceGet]
2023-02-02T04:53:24.400Z CNTRLR   [Node 039] in the process of replying to a NonceGet, won't send another NonceR
                                  eport
2023-02-02T04:53:27.382Z CNTRLR   [Node 039] Timed out while waiting for a response from the node (ZW0201)

... now 189 wants to be really sure that we get its update:
2023-02-02T04:53:41.504Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -60 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.504Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.524Z SERIAL « 0x012300a80001bd1a9f030600babf25a4643aed32314ffb4025ec8e823a22e67c7 (37 bytes)
                                  e6200b764
2023-02-02T04:53:41.526Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.527Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Security2CC_NoSPAN
2023-02-02T04:53:41.527Z CNTRLR » [Node 189] No SPAN is established yet, cannot decode command. Requesting a non
                                  ce...
2023-02-02T04:53:41.567Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00b860      (30 bytes)
2023-02-02T04:53:41.569Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.569Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -72 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.569Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.611Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00c119      (30 bytes)
2023-02-02T04:53:41.612Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.612Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -63 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.612Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.621Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00b36b      (30 bytes)
2023-02-02T04:53:41.622Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.623Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -77 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.623Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.652Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00aa72      (30 bytes)
2023-02-02T04:53:41.653Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.653Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -86 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.654Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.666Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00bb63      (30 bytes)
2023-02-02T04:53:41.667Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.668Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -69 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.668Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.677Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00b169      (30 bytes)
2023-02-02T04:53:41.679Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.679Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -79 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.679Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.688Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00a77f      (30 bytes)
2023-02-02T04:53:41.689Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.690Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -89 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.690Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.709Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00a870      (30 bytes)
2023-02-02T04:53:41.711Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.711Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -88 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.711Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.763Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00a57d      (30 bytes)
2023-02-02T04:53:41.764Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.765Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -91 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.765Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.808Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00a67e      (30 bytes)
2023-02-02T04:53:41.809Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.810Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -90 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.810Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.828Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00bb63      (30 bytes)
2023-02-02T04:53:41.829Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.829Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -69 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.829Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.849Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00a77f      (30 bytes)
2023-02-02T04:53:41.851Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.851Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -89 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.851Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.860Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00a77f      (30 bytes)
2023-02-02T04:53:41.861Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.862Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -89 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.862Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.968Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00a57d      (30 bytes)
2023-02-02T04:53:41.969Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.969Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -91 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.969Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:41.986Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00bc64      (30 bytes)
2023-02-02T04:53:41.988Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:41.988Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -68 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:41.988Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.006Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00bd65      (30 bytes)
2023-02-02T04:53:42.007Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.007Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -67 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.007Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.032Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00c31b      (30 bytes)
2023-02-02T04:53:42.034Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.034Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -61 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.034Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.047Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00c21a      (30 bytes)
2023-02-02T04:53:42.048Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.049Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -62 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.049Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.058Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00ca12      (30 bytes)
2023-02-02T04:53:42.060Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.060Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -54 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.060Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.224Z SERIAL « 0x012300a80001bd1a9f030600babf25a4643aed32314ffb4025ec8e823a22e67c7 (37 bytes)
                                  e6200b764
2023-02-02T04:53:42.225Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.226Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.226Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.248Z SERIAL « 0x011c00a81401bd139f03021917d53b5bc49f4479856c716897000c00b66e      (30 bytes)
2023-02-02T04:53:42.249Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.250Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -74 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.250Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.306Z SERIAL « 0x012300a80001bd1a9f030600babf25a4643aed32314ffb4025ec8e823a22e67c7 (37 bytes)
                                  e6200c112
2023-02-02T04:53:42.308Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.308Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -63 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.308Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.336Z SERIAL « 0x012200a80001bd199f03070066c8f16f45137124602b92dfa63007cc71af65d2d (36 bytes)
                                  b00b726
2023-02-02T04:53:42.338Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:42.338Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Security2CC_NoSPAN
2023-02-02T04:53:42.339Z CNTRLR   [Node 189] No SPAN is established yet, cannot decode command.
2023-02-02T04:53:42.893Z SERIAL « 0x012200a80001bd199f03070066c8f16f45137124602b92dfa63007cc71af65d2d (36 bytes)
                                  b00b726
2023-02-02T04:53:42.894Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:42.895Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:42.895Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:43.303Z SERIAL « 0x012200a80001bd199f03070066c8f16f45137124602b92dfa63007cc71af65d2d (36 bytes)
                                  b00b726
2023-02-02T04:53:43.305Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:43.305Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -73 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:43.305Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:43.327Z SERIAL « 0x011c00a80001bd139f03021917d53b5bc49f4479856c716897000c00a36f      (30 bytes)
2023-02-02T04:53:43.328Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:43.328Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -93 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:43.328Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:43.384Z SERIAL « 0x012200a80001bd199f03070066c8f16f45137124602b92dfa63007cc71af65d2d (36 bytes)
                                  b00c253
2023-02-02T04:53:43.385Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:43.385Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -62 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:43.385Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:43.664Z SERIAL « 0x011c00a80001bd139f03021917d53b5bc49f4479856c716897000c00a36f      (30 bytes)
2023-02-02T04:53:43.665Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:43.665Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -93 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command
2023-02-02T04:53:43.665Z SERIAL » [ACK]                                                                   (0x06)
2023-02-02T04:53:43.997Z SERIAL « 0x011c00a80001bd139f03021917d53b5bc49f4479856c716897000c00a36f      (30 bytes)
2023-02-02T04:53:43.999Z DRIVER   Dropping message with invalid payload
2023-02-02T04:53:43.999Z DRIVER « [Node 189] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -93 dBm
                                  └─[Security2CCMessageEncapsulation] [INVALID]
                                      error: Duplicate command

... more incoming messages ...

... and finally the message reached Node 39 after only 43 seconds:
2023-02-02T04:54:04.569Z DRIVER « [REQ] [SendDataBridge]
                                    callback id:            207
                                    transmit status:        OK, took 43260 ms
                                    repeater node IDs:      89
                                    routing attempts:       2
                                    protocol & route speed: Z-Wave, 40 kbit/s
                                    ACK RSSI:               -66 dBm
                                    ACK RSSI on repeaters:  N/A
                                    ACK channel no.:        1
                                    TX channel no.:         1
```

Similar spamming goes on for a while until there's not enough airtime for the controller to transmit:
```
2023-02-02T04:54:58.392Z DRIVER « [REQ] [SendDataBridge]
                                    callback id:                           215
                                    transmit status:                       Fail, took 0 ms
                                    routing attempts:                      0
                                    protocol & route speed:                Unknown (0x00)
```

I'll reply to your other comment in more detail when I have the time. It surprises me a bit thought that you had a worse experience on 500 series.

**Comment by AlCalzone** [comment]
Anyways, the healing process does stop, so I'm going to close this issue in favor of the other one.

**Comment by bluemantwo** [comment]
I can confirm healing process stops and no residual traffic is seen.  Able to conduct individual heals immediately, further proving the other heal thread is stopped.  Nice!
